models/tasktemplate/fromdir.go

```diff
@@ -31,6 +36,11 @@ func LoadFromDir(dbp zesty.DBProvider, dir string) error {
 		if err := yaml.Unmarshal(tmpl, &tt); err != nil {
 			return fmt.Errorf("failed to unmarshal '%s': '%s'", file.Name(), err)
 		}
+		discoveredTemplates = append(discoveredTemplates, tt)
```

Comment:
Why keep a 2nd collection? You could range over templateimport.GetTemplates

Reply:
templateimport.GetTemplate only returns a list of template names because I want to avoid cyclic import between templateimport, tasktemplate and the step package.
The second collection contains the full complete tasktemplate items.